src/enochecker/checkerservice.py

```diff
@@ -39,15 +39,13 @@
     Required("method", CHECKER_METHODS),  # method to execute
     Required("address", str),  # address to check
     Optional("runId", int, 0),  # internal ID of this run inside our db
-    Optional("team", str, "FakeTeam"),  # the team name
+    Optional("teamName", str, "FakeTeam"),  # the team name
```

Comment:
Is team not clear enough? This is more to type 🤷‍♂️

Reply:
The spec says otherwise though: https://github.com/enowars/EnoEngine/#post-